### ISSUE TITLE
tourism=aquarium

### ISSUE THREAD

**Issue by sorcrosc** [gravitystorm/openstreetmap-carto#2223]
What do you think about rendering [tourism=aquarium](http://wiki.openstreetmap.org/wiki/Tag:tourism%3Daquarium
), at least for the name?

I think it could be treated the same as tourism=zoo

Sample here:
https://www.openstreetmap.org/way/323967826

**Comment by kocio-pl** [comment]
Makes sense for me in general.

**Comment by Jotam** [comment]
I agree. tourism=aquarium should be rendered. Even if it is just with the generic dot.

**Comment by matkoniecz** [comment]
http://wiki.openstreetmap.org/wiki/Tag:tourism=aquarium?uselang=pl - only 113 ways worldwide, in addition there are 132 nodes.

Seems to be a really rare tag.

**Comment by pnorman** [comment]
Not enough uses for a new rendering, and it's not a shop so it wouldn't get a dot.

**Comment by Jotam** [comment]
But why not treat it in the same way as tourism=zoo? Even if it is few instances, it's annoying NOT to find it when you look for it. And it doesn't do any harm to render it, does it?

**Comment by dieterdreist** [comment]
sent from a phone

> Il giorno 16 lug 2016, alle ore 20:39, Paul Norman <notifications@github.com> ha scritto:
> 
> Not enough uses for a new rendering, and it's not a shop so it wouldn't get a dot.
> 


it could be seen as a variant/subtype of zoo and get a name in the same style...

**Comment by dieterdreist** [comment]
sent from a phone

> Il giorno 16 lug 2016, alle ore 20:24, Mateusz Konieczny <notifications@github.com> ha scritto:
> 
> Seems to be a really rare tag.


it's mostly a very significant feature, even if it's not occurring very often it doesn't mean it's a "rare tag" (because there's no alternative tagging method) but rather a rare (but important) feature.

**Comment by sorcrosc** [comment]
> Not enough uses for a new rendering, and it's not a shop so it wouldn't get a dot.
There have been positive feedbacks on treating aquarium like zoo, which seems reasonable (and easy?), and no negative one. Can you comment on that?

**Comment by Jotam** [comment]
@pnorman: Please comment! Simply closing a topic when there is still need for discussion is not the best procedure for a community.

**Comment by Adamant36** [comment]
Any chance of revisiting this? According to Wikipedia and an internet search, there's only between 300 and 500 of them in the world anyway. So there will never be a large amount of the them mapped, but I still its worth rendering since they are major places. It could be either a dot or maybe someone could come up with an icon. A fish with some bubbles would be cool.

**Comment by matkoniecz** [comment]
> there's only between 300 and 500 of them in the world anyway

I am tempted to classify it as result of stupid tagging scheme and continue to ignore it (it should be tourism=attraction attraction=aquarium, unfortunately nobody spotted it).

Something that appears less than 1000 times worldwide should really try to use both one of more general tags and more specific one, to make easier to support it.

And given less than 500 usages worldwide I would not add special icon for that.

**Comment by Adamant36** [comment]
Ok. Just thought id ask. I hadn't considered it might be a miss tagging thing. I still think there might be some things with naturally low numbers worth special rendering, but this could, likely, not be one of them.

**Comment by dieterdreist** [comment]
sent from a phone

> On 4. Jul 2018, at 08:57, Mateusz Konieczny <notifications@github.com> wrote:
> 
> I am tempted to classify it as result of stupid tagging scheme and continue to ignore it (it should be tourism=attraction attraction=aquarium, unfortunately nobody spotted it).


that would be far worse. tourism=attraction is not saying anything meaningful, it is a way to drop a label without caring for semantics, or it is an importance flag when combined with other tags (more important than similar objects without the tag).

It also would not be suitable for aquariums that aren’t attractions in their area.

**Comment by matkoniecz** [comment]
> tourism=attraction is not saying anything meaningful

that is why it would be tourism=attraction + attraction=aquarium, not bare tourism=attraction

> It also would not be suitable for aquariums that aren’t attractions in their area.

tourism=aquarium is not better for that

**Comment by dieterdreist** [comment]
sent from a phone

> On 4. Jul 2018, at 10:07, Mateusz Konieczny <notifications@github.com> wrote:
> 
> tourism=attraction is not saying anything meaningful
> 
> that is why it would be tourism=attraction + attraction=aquarium, not bare tourism=attraction
> 


what would this change for this style? Either you render a label for any attraction (i.e. not specific to the feature, and people could abuse this for label dropping, where questioning the tagging might be difficult (how do you verify something is a tourist attraction?)), or if you check for t=attraction plus attraction=aquarium you have not won anything.

**Comment by matkoniecz** [comment]
> what would this change for this style?

This style (and all other data consumers) would not require special support for tourism=aquarium, support for tourism=attraction would be sufficient.

**Comment by dieterdreist** [comment]
you can propose it on the tagging mailing list, but attraction currently is mostly used for animals in captivity and attractions in theme parks, like roller coasters https://taginfo.openstreetmap.org/keys/attraction#values

**Comment by polarbearing** [comment]
On the wiki zoo page, it explicitly recommends to use _tourism=aquarium_ and not a zoo subtype. 
Zoo is rendered with its name without icon, it would not hurt to do the same for aquarium. 
colour=tourism brown

**Comment by polarbearing** [comment]
tourism=aquarium 683x
attraction=aquarium 12x
aquarium=yes 11x
https://wiki.openstreetmap.org/wiki/Tag%3Atourism%3Daquarium - approved by voting 2014

**Comment by polarbearing** [comment]
When this issue was discussed in 2016, the usage was cited with 113w+132n = 243 taggings.
When we have now about 700 mapped, that is probably all aquariums that exist worldwide. 

@IVAN-RSM in #3691 - the reason we don't render a symbol for a zoo is that it has a scalable label, and we don't know yet how to handle symbols in such cases, see #3284. 

Currently aquarium is rendered with its building label, if it is mapped as a building, see https://www.openstreetmap.org/way/18330093 as example. 

It would not hurt to label any aquarium (node or way, independent of the building label) with the tourism colour like a zoo. Label scaling would not be necessary since an aquarium is rarely a large campus.

**Comment by kocio-pl** [comment]
> When we have now about 700 mapped, that is probably all aquariums that exist worldwide.

I am not sure if this is all - look at this line:

![taghistory 49](https://user-images.githubusercontent.com/5439713/53293222-1c5b9500-37d0-11e9-9f67-f772021aac4e.png)

I have heard interesting idea to estimate how complete is tagging for a given type of objects (probably o some SotM video): when it comes closer to 100%, the curve is asymptotically getting flat, and in last 2 years it looks more steep.

**Comment by Adamant36** [comment]
> I have heard interesting idea to estimate how complete is tagging for a given type of objects (probably on some SotM video): when it comes closer to 100%, the curve is asymptotically getting flat, and in last 2 years it looks more steep.

I think there was a blog post or diary entry awhile back about shop mapping in the UK and how's its proportional to the amount of people mapping them. If I remember correctly it was based on theory about how new species of bugs in the amazon are found at the rate of entomologist looking for them. So you can't use the current number of known bug species as an indicator of how many actual bug species are out there. I could see how the same logic would apply to tagging. It probably tends to tapper off at the end because as un-mapped items becomes more sparse and harder to find mappers lose interest and move on to other things. Or they have just reached an "I'm done mapping this particular feature" threshold collectively. 

As far as aquariums goes, on Wikipedia it says there are over 200, but its not more specific. Whereas the Association of Zoo's and Aquariums, whoever they are, says there are 233 aquarium's accredited through them. So my personal guess based on that is that there's probably more then 200 but less then 1000. Some places tagged as aquariums in OSM might be addons to zoos or something like that also. So the tagging numbers for them could be slightly over inflated or just more broad then normal.

**Comment by matkoniecz** [comment]
> and how's its proportional to the amount of people mapping them

Yes, flattening of curve may mean that people run out of things to map or that OSM run out of people mapping things.

But steep curve indicates that neither of this happened.

**Comment by imagico** [comment]
Use is world wide but for a relatively large variety of things.

Most uses seem to be for some public display of aquatic life, but that in some regions includes places where display is related to sale as pets (and sometimes also as food).

Use of the tag on polygons seems to be predominantly on individual buildings featuring tanks/pools with aquatic life.  Outdoor pools/ponds are also tagged quite a bit.  The tag is rarely used for larger facilities with multiple buildings and/or outdoor pools.  That is probably partly because most larger facilities do not exclusively feature water tanks with aquatic life but also other exhibits/attractions and will therefore be tagged accordingly.

Documentation on the wiki so far does not provide any guidance regarding semantic delineation of the tag towards others.

Bottom line: Even if we wanted to render the tag now (which i am not sure we would) - it would not be quite clear what kind of use the rendering should support.  Given use of the tag in a similar fashion as `tourism=zoo` (i.e. for larger facilities consisting of multiple buildings/pools) is quite rare rendering it the same way does not seem a good idea.

**Comment by imagico** [comment]
Again - although i have explained this too many times already:  We aim to create a map for the world wide OSM community.  And we do so based on looking at how tags are actually used and where this tag use is reasonably consistent world wide we look at how we can support such with a suitable rendering.  Discussion of ought-to-be tagging from some subjective perspective on what is and what is not correct use of tags that does not derive from observations in the data is not our concern.  Suggesting design ideas that target a tag use concept that applies to maybe a few percent of actual uses of the tag is decidedly not helpful.

If you want to see this rendered you should:

* look at the data
* identify patterns in mapping that are consistent globally across cultural and geographic differences
* document these patterns publicly and provide guidance to mappers how to map this way that does not depend on a certain cutural predisposition
* develop a rendering paradigm that provides specific targeted positive support for the consensus tag use and that fits consistently into our overall design paradigm (if necessary adjusting existing design)

**Comment by matkoniecz** [comment]
https://community.openstreetmap.org/t/tagging-of-aquariums/102370/7

> [This map](https://map.atownsend.org.uk/maps/map/map.html#18/54.30103/-0.40991/H) will show aquariums in the UK and IE mapped as tourism=aquarium as a general tourist thing. It ahould perhaps show larger ones a bit sooner, but it does show them.
> When deciding whether to show them I did look at tag usage and found that it wasn’t a major problem.

**Comment by imagico** [comment]
Thanks for the pointer.  The UK and Ireland account for 55 of 1399 features with the tag, almost all of the polygons there are also tagged as buildings.

Globally, the big users of the tag are the US (187), India (177) and Japan (136).

**Comment by dieterdreist** [comment]
there is also small use of historic=aquarium
https://www.openstreetmap.org/way/23502331
